build.gradle

```diff
@@ -7,13 +7,22 @@
  */
 
 plugins {
+  //id 'java' //remove/add as needed
   id "net.saliman.properties" version "1.4.6"
-  id "com.marklogic.ml-gradle" version "3.10.1"
+  id "com.marklogic.ml-gradle" version "3.13.0"
 }
 
 repositories {
-  jcenter()
-  maven { url "http://developer.marklogic.com/maven2/" }
+  // To use gradle in offline mode, you just need to add the --offline option. E.g. gradle --offline mlDeploy
+  if (project.getGradle().startParameter.isOffline()) {
+    println "Using offline repositories"
+    maven { url uri(projectMavenRepo) }
+  } else {
+    println "Using online repositories"
+    jcenter()
+    mavenCentral()
+    maven { url "http://developer.marklogic.com/maven2/" }
+  }
```

Comment:
this bit needs to be in buildscripts I think, and only there seems enough. Perhaps in case you need to compile something...

Reply:
Nope, you need both places. Not entirely clear what serves what exactly. buildscripts seems to cater task deps, repositories and dependencies seems to cater compilation of code and such..